server/src/main/java/org/elasticsearch/action/search/SearchQueryThenFetchAsyncAction.java

```diff
@@ -126,6 +130,22 @@ protected SearchPhase getNextPhase(final SearchPhaseResults<SearchPhaseResult> r
     }
 
     private ShardSearchRequest rewriteShardSearchRequest(ShardSearchRequest request) {
+        if (request.source() != null && request.source().queries().isEmpty() == false) {
+            BoolQueryBuilder boolQueryBuilder = new BoolQueryBuilder();
```

Comment:
We probably not need to rewrite to `bool` if there is only 1 searchQueryWrapperBuilder, but also I assume this would be quite a rare case.

Reply:
Following up on the comment above, maybe we it does make sense to ensure that queries is only used with at least 2 of either queries or knn searches? So if there's only one `SearchQueryWrapper` it would also require that there's at least one knn search.

Reply:
This is no longer possible with the removal of default linear combination for multiple queries.